src/app/pages/review-attributes/review-attributes.page.ts

```diff
@@ -1,60 +1,55 @@
-import { Component, OnInit } from "@angular/core";
-import { AlertService } from "../../_services/alert/alert.service";
+import { Component, OnInit } from '@angular/core';
+
 import { ReviewAttributesModelService } from "./_services/review-attributes.model.service";
 
+import { AlertService } from '../../_services/alert/alert.service';
+
 @Component({
-  selector: "app-review-attributes",
-  templateUrl: "./review-attributes.page.html",
-  styleUrls: ["./review-attributes.page.scss"],
+  selector: 'app-review-attributes',
+  templateUrl: './review-attributes.page.html',
+  styleUrls: ['./review-attributes.page.scss'],
 })
+
 export class ReviewAttributesPage implements OnInit {
-  headerPageTitle: string = "Review Attributes";
 
+  headerPageTitle: string = 'Review Attributes';
   phraseToBeReviewed: String = "";
   getNextPhraseButtonDisabled: boolean = false;
   approveAndRejectButtonDisplayed: boolean = false;
+  reasonIDApprove: number = 1;
```

Comment:
This is only used in one method. You could define it in that method, following the general rule that it is best to define variables near where they are first used.